core/source.ts

```diff
@@ -1,13 +1,13 @@
-import { basename, dirname, extname, join } from "../deps/path.ts";
+import { Data, Directory, Event, Loader, Page, Site, Source } from "../core.ts";
 import { existsSync } from "../deps/fs.ts";
+import { basename, dirname, extname, join, relative } from "../deps/path.ts";
 import { SiteDirectory, SitePage } from "./filesystem.ts";
 import {
   concurrent,
   Exception,
   normalizePath,
-  searchByExtension,
+  searchByExtension
 } from "./utils.ts";
-import { Data, Directory, Event, Loader, Page, Site, Source } from "../core.ts";
```

Comment:
Auto import sorter made it's magic here, sorry if that's annoying. It's sorting imports alphabetically by the imported path.